### ISSUE TITLE
Sentinel passwords logged in agent.log

### ISSUE THREAD

**Issue by vinsampath-hw-it** [DataDog/integrations-extras#2053]
https://github.com/DataDog/integrations-extras/blob/d046b59f7b93ce0f1439f7ec7c5562c920248576/redis_sentinel/datadog_checks/redis_sentinel/redis_sentinel.py#L31

Sentinel passwords logged in agent.log. Submitting a request to review and mask it or stop logging it.

**Comment by vinsampath-hw-it** [comment]
Possible Fix

```
def check(self, instance):
    sanitized_instance = dict(instance)
    sanitized_instance.pop('sentinel_password', None)  # Remove 'sentinel_password' key

    self.log.info(sanitized_instance)
```

**Comment by alopezz** [comment]
This seems to have been solved in #2059.
